test/inspector/test-inspector-ip-detection.js

```diff
@@ -22,7 +22,7 @@ function checkListResponse(instance, err, response) {
   const match = wsUrl.match(/^ws:\/\/(.*):9229\/(.*)/);
   assert.strictEqual(ip, match[1]);
   assert.strictEqual(res['id'], match[2]);
-  assert.strictEqual(ip, res['devtoolsFrontendUrl'].match(/.*ws=(.*):9229/)[1]);
+  assert.strictEqual(ip, res['devtoolsFrontendUrl'].match(/ws=(.*):9229/)[1]);
```

Comment:
Oh, sorry, I've missed this part can take up similar parts.

However, this RegExp checks a URL like:
`chrome-devtools://devtools/bundled/inspector.html?experiments=true&v8only=true&ws=192.168.137.2:9229/151fc61d-6045-4692-b651-58bb5ba2c83d`

If there is any chance this URL can have two `ws=` parts, let me know and I will remove this commit.

cc @nodejs/v8-inspector